### ISSUE TITLE
Using an old smartphone as webcam/camera recorder.

### ISSUE THREAD

**Issue by hellishvictor** [Genymobile/scrcpy#5345]
Hi, here I have and old Huawei P Smart 2019 smartphone with Android 11 that I want to use as webcam; I just open its camera, select video, and after a minute the screen turn black with a line showing "Press the screen for active the camera", but I want that it stay on. Does anybody know how to fix that?
Cheers.

**Comment by rom1v** [comment]
It's your camera application, isn't it? Try another camera app (scrcpy could not capture the camera directly on Android < 12).

**Comment by hellishvictor** [comment]
Yes, I know the "--video-source=camera" option doesn't work with Android 11, but that's not is what I looking for since it don't show the record, zoom, etc button that are the default camera application.
